### DISCUSSION TITLE
Custom Page with Livewire component crashes after update to v3.0.13

### DISCUSSION THREAD

**Discussion by demobiel** [filamentphp/filament#7585] [Help]
We have a custom page with a Livewire component. 

EventSettings.php (custom page)
```
class EventSettings extends Page
{
    protected static string $view = 'filament.event.pages.event-settings';
    
}
```
event-settings.blade.php
```
<x-filament-panels::page>

    @livewire('products-component')

</x-filament-panels::page>
```
ProductsComponent.php
```
class ProductsComponent extends Component
{

    public function render(): View
    {
        return view('livewire.products-component');
    }
}
```
products-component.blade.php
```
<div>
        this is my product
</div>
```

The bug we receive is: 
```
Livewire page component layout view not found: [components.layouts.app]
```
When we remove the  `@livewire('products-component')` from the blade file of the custom page everything works fine.

I don't understand why this happens, or why a "subcomponent" should require a base layout.
Is this a filament or livewire bug as yesterday Livewire released v3.0.0-beta-8? 
Does somebody know a workaround?

**Comment by nikcani** [comment]
this seems to be the same error as mentioned here https://github.com/filamentphp/filament/discussions/7554

**Reply by nikcani** [reply]
@demobiel a possible workaround right now is to add `"livewire/livewire": "3.0.0-beta.7"` to your composer.json

**Comment by demobiel** [comment]
Here is a bug report on the livewire side: https://github.com/livewire/livewire/discussions/6281

They mention another possible workaround: create an empty blade file.

**Reply by demobiel** [reply]
which doesn't work, so indeed at the moment stay on the version 3.0.0-beta.7

**Comment by danharrin** [comment]
A few hours ago I released a version of Filament that locked Livewire to not use that version, so hopefully it shouldnt be an issue until that is resolved
